.vscode/launch.json

```diff
@@ -15,7 +43,9 @@
 			"request": "launch",
 			"name": "Launch Test",
 			"program": "${workspaceFolder}/test.js",
-			"args": ["--local-debug"]
+			"args": [
+				"--local-debug"
+			]
 		}
 	]
-}
+}
```

Comment:
A new line is needed to eliminate the warning here. Ref: https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file